munk/seed.py

```diff
@@ -1,11 +1,15 @@
 import yaml
-from insert import insert
+from db import DB
 
 
-with open('seed.yaml', 'r') as file:
-    db = yaml.load(file.read())
+def seed_db(seed_file_name, db_file_name):
+    with open(seed_file_name, 'r') as file:
+        seed = yaml.load(file.read())
```

Comment:
Couldn't this be `seed = yaml.load(file)` ?

Reply:
Fixed.